noaa.go

```diff
@@ -18,6 +18,10 @@ const (
 	APIAccept = "application/ld+json"       // Changes may affect struct mappings below
 )
 
+// UserAgent provides the User-Agent header value used for API request.
+// This defaults to the APIKey const value.
+var UserAgent = APIKey
```

Comment:
Sorry for the delay. This is a good idea. I think it would be good to put this behind a setter, even though indirection might be overkill short-term. 

Kind of like we do with units. Something along the lines of `noaa.SetUserAgent(<string>)` and there set your userAgent variable. I think everything else you changed works as-is. Rename the variable to `userAgent` so it's not exported. This gives us a little more flexibility later to change how this works and to validate it as needed. Thanks again.

Reply:
Alrighty, done!